<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamingWidget</class>
 <widget class="QWidget" name="StreamingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>237</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>417</width>
    <height>237</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>417</width>
    <height>237</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>youyuan</family>
    <pointsize>-1</pointsize>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>StreamingWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:youyuan;
font-size:16px;
color:rgb(255,255,255);
}

QWidget#widget,QWidget#widget2{
background-color:rgb(0,0,0);
}

QPushButton{
background-color:rgb(61,61,61);
border-radius:10px;
}

QPushButton:hover{
background-color:rgb(45,45,45);
}


/*滑竿*/
QSlider::groove:horizontal {
    border: 1px solid #4A708B;
    background: #C0C0C0;
    height: 3px;
    border-radius: 2px;
    padding-left:-1px;
    padding-right:-1px;
}
/*已经划过的从地方*/
QSlider::sub-page:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #B1B1B1, stop:1 #c4c4c4);
    background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
        stop: 0 #5DCCFF, stop: 1 #1874CD);
    border: 1px solid #4A708B;
    border-radius: 2px;
}
/*还没有滑上去的地方*/
QSlider::add-page:horizontal {
    background: #575757;
    border: 0px solid #777;
    border-radius: 2px;
}
/*中间滑块*/
QSlider::handle:horizontal
{
    /*背景颜色设置为辐射聚焦*/
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5,
    stop:0.6 #45ADED, stop:0.8 rgba(255, 255, 255, 255));
    
    /*形成圆*/
    width: 8px;
    border-radius: 4px;
    
    /*上沿、下沿超出滑竿*/
    margin-top: -3px;
    margin-bottom: -2px;
}

QSlider::handle:horizontal:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 #2A8BDA,
    stop:0.8 rgba(255, 255, 255, 255));

    /*形成圆*/
    width: 8px;
    border-radius: 4px;
    
    /*上沿、下沿超出滑竿*/
    margin-top: -3px;
    margin-bottom: -2px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labRtmp">
        <property name="font">
         <font>
          <family>youyuan</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>RTMP推流地址:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtRtmp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{ 
border-radius:5px;
background:rgb(61,61,61);
font-family:Comic Sans MS;
}</string>
        </property>
        <property name="text">
         <string>rtmp://192.168.239.180/live</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item row="0" column="0" colspan="6">
       <widget class="Slider" name="slider">
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QWidget" name="widget4" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="Button" name="btnSmooth">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Res/icon/BeautyFace/smooth.png</normaloff>Res/icon/BeautyFace/smooth.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="Button" name="btnWhite">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Res/icon/BeautyFace/white.png</normaloff>Res/icon/BeautyFace/white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="Button" name="btnThin">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Res/icon/BeautyFace/thin.png</normaloff>Res/icon/BeautyFace/thin.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="Button" name="btnBrightness">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Res/icon/BeautyFace/white.png</normaloff>Res/icon/BeautyFace/white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="Button" name="btnContrast">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Res/icon/BeautyFace/contrast.png</normaloff>Res/icon/BeautyFace/contrast.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="Button" name="btnSaturation">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Res/icon/BeautyFace/saturation.png</normaloff>Res/icon/BeautyFace/saturation.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labSmooth">
           <property name="font">
            <font>
             <family>youyuan</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>磨皮</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labWhite">
           <property name="font">
            <font>
             <family>youyuan</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>美白</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labThin">
           <property name="font">
            <font>
             <family>youyuan</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>去噪</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="labBrightness">
           <property name="font">
            <font>
             <family>youyuan</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>亮度</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="labContrast">
           <property name="font">
            <font>
             <family>youyuan</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>对比度</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="labSaturation">
           <property name="font">
            <font>
             <family>youyuan</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>饱和度</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="6">
       <widget class="QWidget" name="widget3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="Button" name="btnCtrl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(120, 120,120, 255));
 border: none;
border-radius:15px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }
QPushButton:!hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(39, 39, 39, 255), stop:0.495 rgba(120, 120, 120, 255), stop:0.505 rgba(120,120, 120, 255), stop:1 rgba(39, 39,39, 255));
 border: none;
border-radius:15px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }</string>
           </property>
           <property name="text">
            <string>开始直播</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>Button.h</header>
  </customwidget>
  <customwidget>
   <class>Slider</class>
   <extends>QSlider</extends>
   <header>Slider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
